cmd/signaling/main.go

```diff
@@ -36,12 +36,6 @@ func main() {
 	}
 
 	store := stores.NewMemoryStore()
-	go func() {
-		for range time.NewTicker(1 * time.Minute).C {
-			count := store.DebugTotalLobbyCount()
-			logger.Info("store length", zap.Int("count", count))
-		}
-	}()
```

Comment:
This is not needed anymore.

Reply:
I would have left it in there until we are happy with the reslt, nice debugging tool

Reply:
I looked at it and didn't find it very useful anymore. We should really switch to a non in-memory store instead.